### DISCUSSION TITLE
Trouble with Teleports

### DISCUSSION THREAD

**Discussion by eauxnine** [cinderblocks/radegast#22] [Q&A] [answered]
Since I updated the viewer, I cannot teleport anywhere on this specific viewer when offered. Is this a known issue? I get the error of disconnected Sim Shutdown. Teleporting to Landmarks aren't an issue.

**Comment by ghost** [comment]
I have the same problem. Teleports simply dont work. THe stop with "Disconnected from gid" even tha avatar is still loogen in and can walk around, after the failed teleport

**Comment by cinderblocks** [comment, marked as answer]
Please follow this issue in the bug report here: https://github.com/cinderblocks/radegast/issues/23
